Sources/SwiftCliCore/ChartConstructor.swift

```diff
@@ -94,7 +111,9 @@ public class ChartConstructor {
         default:
             var middleStitches = ""
             for stitch in row {
-                middleStitches += "\(stitchPrinting[stitch] ?? "")│"
+                if let getStitch = allowedStitchesInfo.first(where: { $0.name == stitch }) {
```

Comment:
I remember you repeat this `allowedStitchesInfo.first(where: { $0.name == stitch })` logic elsewhere, so it seems like that would be useful to have as a function. Think about where the best place for such a function would be.

Reply:
I decided to add it in StitchLibrary as `stitchLookup` which will return the corresponding `StitchInfo` object, and then I can get whatever value I need from that as necessary in the context of the program (either how to print the stitch's symbol or what increase or decrease value the stitch has)